x-pack/filebeat/module/misp/threat/test/misp-test.json.log-expected.json

```diff
@@ -0,0 +1,47 @@
+[
+    {
+        "@timestamp": "2017-03-30T12:54:26.000Z",
+        "event.category": "threat-intel",
+        "event.dataset": "misp.threat",
+        "event.module": "misp",
+        "event.type": "indicator",
+        "fileset.name": "threat",
+        "input.type": "log",
+        "log.offset": 0,
+        "service.type": "misp",
+        "threat.threat_indicator.attack_pattern": "[destination.ip = \"98.235.162.24\"]",
```

Comment:
The MISP stuff should all be nested under `misp.`, as this is for a MISP module and hasn't been normalized across other potential sources of IOCs.

Adding "threat" directly at the root here would also be confusing vs the upcoming support for "threat" in ECS (https://github.com/elastic/ecs/pull/505).

Note that when "threat" lands in ECS, I would definitely see some of the fields in this module using the canonical ECS fields. As such, if you have thoughts on https://github.com/elastic/ecs/pull/505, your feedback is welcome :-)

Reply:
OK, I will move them under misp for now.